Untitled Game/src/Levels/LevelWithRoomsMockup.gd

```diff
@@ -0,0 +1,80 @@
+extends Node2D
+
+var _LABEL_POSITION_OFFSET = 100
+
+var _lockCamera = false
+var _startCountDown = 5
+var _timerOn = false
+
+onready var camera = $Camera2D
+onready var player = $LirikYaki
+onready var firstLockArea = $FirstRoomLockArea
+onready var cameraManager = CameraManager.new(player, camera)
+onready var label = $CanvasLayer/Label
+onready var leftFirstRoomCollider = $FirstRoomBattleBoundary/LeftSide
+onready var rightFirstRoomCollider = $FirstRoomBattleBoundary/RightSide
+
```

Comment:
put these in the ready method instead and make sure they exist, if they don't throw an exception with the pertinent information to what is missing from the scen

Reply:
I could be wrong, but i think the onready keyword is a shortcut for doing that. 

[https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html#onready-keyword](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html#onready-keyword)

If im wrong than ill go into correcting this ASAP.

Reply:
Fixed in commit 3ef58d06728a512bdf39347e03dfe72dee807f0f.